webui/react/src/hooks/useSettings.test.tsx

```diff
@@ -8,7 +8,6 @@ import { CompatRouter } from 'react-router-dom-v5-compat';
 import StoreProvider from 'contexts/Store';
```

Comment:
will we keep using useContext?
If so, I think we cannot resolve rendering issues